packages/bundles/kairos-overlay-files/collection.yaml

```diff
@@ -0,0 +1,8 @@
+packages:
+  - name: "kairos-overlay-files"
+    category: "bundles"
+    version: "2.4.0"
```

Comment:
how we ganno manage this version? I guess independent from the kairos repo rigth? Its just a match for now

Reply:
just saw below....I would set this to 1.0.0 and leave it unconnected to the kairos version for now, as bumps to this package do not neccesarily translate to the kairos version no?

Reply:
yes, I wasn't sure either. My plan was to start from what kairos is now and let it diverge. But I think your suggestion is better. Let's start with 1.0.0 to make it clear that they have no relation.